spec/game_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'game'
+
+describe Game do
+  let(:game) { Game.new }
+
+  describe '#initialize' do
+    it 'it initializes with no rolls played' do
+      expect(game.rolls).to eq []
+    end
+  end
+
+  describe '#roll' do
+    it { is_expected.to respond_to(:roll).with(1).argument }
+
+    it 'it can roll and knock down 10 pins' do
+      game.roll(10)
+      expect(game.roll(10)).to include 10
```

Comment:
i think this should be expect(game.rolls) instead of expect(game.roll) because your array has an 's' in the end